components/layout/DocsLayout.js

```diff
@@ -82,23 +82,26 @@ export default function DocsLayout({ post, navItems = {}, children }) {
                 </button>
               </div>
             )}
-            
+
             <div className={`xl:flex ${post.toc && post.toc.length ? 'xl:flex-row-reverse' : ''}`}>
               <TOC toc={post.toc} depth={3} className="bg-blue-100 mt-4 p-4 sticky top-20 overflow-y-auto max-h-screen xl:bg-transparent xl:mt-0 xl:pb-8 xl:w-72" />
               <div className="px-4 sm:px-6 xl:px-8 xl:flex-1 xl:max-w-184">
               <Heading level="h1" typeStyle="heading-lg">
                 {post.title}
               </Heading>
             {
-              post.isPrerelease 
-              ? <h3 className="text-lxl font-normal text-gray-800 font-sans antialiased">To be released on {post.releaseDate}</h3> 
+              post.isPrerelease
+              ? <h3 className="text-lxl font-normal text-gray-800 font-sans antialiased">To be released on {post.releaseDate}</h3>
               : null
             }
             <div>
               <p className="text-sm font-normal text-gray-600 font-sans antialiased">
-                Found an error? Have a suggestion? 
+                Found an error? Have a suggestion?
                 {generateEditLink(post)}
               </p>
+              <p className="text-center text-sm font-normal text-gray-600 font-sans antialiased">
+                The <a href="https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md">human-readable markdown file<a/> is the source of truth for the specification.
```

Comment:
I agree with @smoya here. This will be displayed on every subpage of the documentation and not only where the specification is rendered. Such information should be added to the markdown in the spec repository, not manually updated in website.